charts/common/values.yaml

```diff
@@ -363,6 +363,11 @@ cloudsqlProxy:
     #instanceConnectionName: project_name:region:instance_name
     port: 3306
     secretName: migration
+    # Setting up Time to live Strategy for the migration template workflow, if user don't specify any by default will be 30 days max
+    ttlStrategy:
+      secondsAfterCompletion: 2592000 # Time to live after workflow is completed, replaces ttlSecondsAfterFinished
```

Comment:
Can you specify somewhere here this is 1 month

Reply:
@ofrivera yeah actually the comment above is stating that but if you think is more useful I will mention that on each line